pkg/apis/migration/migration.go

```diff
@@ -407,3 +417,61 @@ func (volumeMigration *volumeMigration) registerVolume(ctx context.Context, volu
 	}
 	return volumeID.Id, nil
 }
+
+// cleanupStaleCRDInstances helps in cleaning up stale volume migration CRD instances
+func (volumeMigration *volumeMigration) cleanupStaleCRDInstances(ctx context.Context) {
+	log := logger.GetLogger(ctx)
```

Comment:
you can remove this log and also let's not supply `ctx` to this routine method.

Reply:
Makes sense